sdk/wallet-adapter/adapters/wallet-standard-adapter/src/index.ts

```diff
@@ -17,15 +17,21 @@ type Events = {
 
 export { StandardWalletAdapter };
 
+// These are the default features that the adapter will check for:
+export const DEFAULT_FEATURES: (keyof StandardWalletAdapterWallet["features"])[] =
+  ["sui:signAndExecuteTransaction"];
+
 export class WalletStandardAdapterProvider implements WalletAdapterProvider {
   #wallets: Wallets;
   #adapters: Map<StandardWalletAdapterWallet, StandardWalletAdapter>;
   #events: Emitter<Events>;
+  #features: string[];
 
-  constructor() {
+  constructor({ features }: { features?: string[] } = {}) {
     this.#adapters = new Map();
-    this.#wallets = DEPRECATED_getWallets();
+    this.#wallets = getWallets();
```

Comment:
This is an intentional change, the old deprecated wallet interface has been gone for quite a while, so this should be safe to remove